### PR TITLE
bump kerlescan to v 0.55

### PR DESCRIPTION
**Pull request by eherget**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by sthirugn** [comment]
> ## Secure Coding Practices Checklist Link
> 
>     * https://github.com/RedHatInsights/secure-coding-checklist
> 
> 
> ## Secure Coding Checklist
> 
>     * [ ]  Input Validation
> 
>     * [ ]  Output Encoding
> 
>     * [ ]  Authentication and Password Management
> 
>     * [ ]  Session Management
> 
>     * [ ]  Access Control
> 
>     * [ ]  Cryptographic Practices
> 
>     * [ ]  Error Handling and Logging
> 
>     * [ ]  Data Protection
> 
>     * [ ]  Communication Security
> 
>     * [ ]  System Configuration
> 
>     * [ ]  Database Security
> 
>     * [ ]  File Management
> 
>     * [ ]  Memory Management
> 
>     * [ ]  General Coding Practices

I think you are supposed to check these items in the checklist to make sure that you reviewed them when you submitted the PR.

File: Pipfile [hunk 1]
```diff
@@ -28,10 +28,10 @@ flask-sqlalchemy = "*"
 alembic = "*"
 flask-script = "*"
 flask-migrate = "*"
-kerlescan = {editable = true, git = "https://github.com/RedHatInsights/kerlescan.git", ref = "0.50"}
 bitmath = "*"
 jsonpatch = "*"
 insights-core = "*"
+kerlescan = {editable = true, ref = "0.55", git = "https://github.com/RedHatInsights/kerlescan.git"}
```
Comment (sthirugn) [inline Pipfile:34]: any reason for changing the order of these fields (editable, ref, git) from (editable, git, ref) previously which was in alphabetical order.

**Review by sthirugn — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (eherget) [reply Pipfile:34]: I used the `pipenv uninstall kerlescan` command, followed by `pipenv install --editable git+https://github.com/RedHatInsights/kerlescan.git@0.55#egg=kerlescan` command to get pipenv to install the correct version of kerlescan into my env.  Without doing this, regardless of running `pipenv update`, `pipenv lock`, `pipenv install --dev`, etc, running the unit tests would complain about not finding the kerlescan module.
The `pipenv install ...` command put the args in this order and the entry in the position to where it was moved.

**Comment by eherget** [comment]
> > ## Secure Coding Practices Checklist Link
> > ```
> > * https://github.com/RedHatInsights/secure-coding-checklist
> > ```
> > 
> > 
> > ## Secure Coding Checklist
> > ```
> > * [ ]  Input Validation
> > 
> > * [ ]  Output Encoding
> > 
> > * [ ]  Authentication and Password Management
> > 
> > * [ ]  Session Management
> > 
> > * [ ]  Access Control
> > 
> > * [ ]  Cryptographic Practices
> > 
> > * [ ]  Error Handling and Logging
> > 
> > * [ ]  Data Protection
> > 
> > * [ ]  Communication Security
> > 
> > * [ ]  System Configuration
> > 
> > * [ ]  Database Security
> > 
> > * [ ]  File Management
> > 
> > * [ ]  Memory Management
> > 
> > * [ ]  General Coding Practices
> > ```
> 
> I think you are supposed to check these items in the checklist to make sure that you reviewed them when you submitted the PR.

I think we are, too.  But this was just a version change of a dependency.  There was no code in the PR.  Plus I'm trying to get back to focusing on FedRAMP stuff and this detour keeps getting longer!

File: Pipfile (hunk 1 above)
Comment (sthirugn) [reply Pipfile:34]: Interesting, I didn't know about it. Thank you for the explanation.

**Comment by sthirugn** [comment]
> I think we are, too. But this was just a version change of a dependency. There was no code in the PR. Plus I'm trying to get back to focusing on FedRAMP stuff and this detour keeps getting longer!

I went ahead and checked all the items on the checklist. This is part of the process and please use it in the future

**Comment by eherget** [comment]
> > I think we are, too. But this was just a version change of a dependency. There was no code in the PR. Plus I'm trying to get back to focusing on FedRAMP stuff and this detour keeps getting longer!
> 
> I went ahead and checked all the items on the checklist. This is part of the process and please use it in the future

From the doc linked at the top of the description - https://github.com/RedHatInsights/secure-coding-checklist

"Only pay attention to the sections that pertain to code. For example if you wrote something only have to do with databases, only look at the database portion. There is no need to read about file management unless your code does file management. "

**Review by CodeHeeler — APPROVED** [review]

**Review by sthirugn — APPROVED** [review]

**Pull request merged**